src/ReCaptchaLogger.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace DS\Library\ReCaptcha;
+
+use Psr\Log\LoggerAwareInterface;
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
+use Psr\Log\InvalidArgumentException;
+use Psr\Log\NullLogger;
+
+class ReCaptchaLogger implements LoggerAwareInterface
+{
+    private $logger;
+    
+    public function __construct()
```

Comment:
Constructor dependency is preferred:
More useful in terms of static analysis, etc...

Setters injection (mostly for optional dependencies):
Routing :  RouterFactory -> 

module1:
routes

module2:
routes

addRoutes(routes) ->
  each modules
     factory->addRoutes(module->getRoutes())